_opensearch/rest-api/document-apis/multi-get.md

```diff
@@ -0,0 +1,143 @@
+---
+layout: default
+title: Multi-get document
+parent: Document APIs
+grand_parent: REST API reference
+nav_order: 20
+---
+
+# Multi-get documents
+
+Introduced 1.0 {: .label .label-purple }
+
+You can use the multi-get operation to get back multiple documents from one index or multiple indices in one request.
+
```

Comment:
Maybe we can connect it to the get API and say here that mget lets you combine multiple get operations in one request.